divi/src/BlockMemoryPoolTransactionCollector.cpp

```diff
@@ -16,9 +16,10 @@
 #include <Settings.h>
 extern Settings& settings;
 
+extern CChain chainActive;
 extern CCoinsViewCache* pcoinsTip;
 
-bool IsFinalTx(const CTransaction& tx, int nBlockHeight = 0 , int64_t nBlockTime = 0);
+bool IsFinalTx(const CTransaction& tx, const CChain& chainActive_, int nBlockHeight = 0 , int64_t nBlockTime = 0);
```

Comment:
May I suggest the alias 'activeChain' instead of 'chainActive_' to try to mitigate name collisions here? :P

Reply:
Good point, changed.